File: MidTerm/Count_Me_3.java
import java.util.Scanner;
class Count_Me_3{
    public static void main(String args[]){
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        sc.nextLine();
        for(int i=0; i<t; i++){
            String str=sc.nextLine();
            int cap=0, small=0, num=0;
            for(int j=0; j<str.length(); j++){
                if(str.charAt(j)>='A' && str.charAt(j)<='Z') cap++;
                if(str.charAt(j)>='a' && str.charAt(j)<='z') small++;
                if(str.charAt(j)>='0' && str.charAt(j)<='9') num++;
            }
            System.out.println(cap+" "+small+" "+num);
        }
        sc.close();
    }
}

File: MidTerm/Count_Me_3.c
#include <stdio.h>
int main()
{
    int t;
    scanf("%d", &t);
    for (int i = 0; i < t; i++)
    {
        char str[10000];
        scanf("%s", str);
        int cap=0, small=0, num=0;
        for(int j=0; str[j]!='\0'; j++){
            if(str[j]>='A' && str[j]<='Z') cap++;
            if(str[j]>='a' && str[j]<='z') small++;
            if(str[j]>='0' && str[j]<='9') num++;
        }
        printf("%d %d %d\n", cap, small, num);
    }
    return 0;
}